pkgs/development/libraries/qt-6/modules/qtbase/default.nix

```diff
@@ -263,6 +263,8 @@ stdenv.mkDerivation rec {
       "-DQT_FEATURE_system_sqlite=ON"
       "-DQT_FEATURE_openssl_linked=ON"
       "-DQT_FEATURE_vulkan=ON"
+      # https://bugreports.qt.io/browse/QTBUG-136060
```

Comment:
Can you maybe add a small comment here that explains why we're doing this, just so people don't have to go read the bug tracker? Just one line like "don't leak OS version into the final output"